.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "website/themes/docsy"]
+	path = website/themes/docsy
+	url = https://github.com/google/docsy.git
```

Comment:
I know the hugo examples all use submodules for the themes. But they're really annoying. We can try it but I'd almost rather just copy in the theme and deal with updates by hand if we ever need to update it from the upstream.

Reply:
I share your irritation with submodules and have been considering alternatives, but I haven't found a better solution. The biggest concern is that the docsy theme has a lot of files, and I don't think we want to carry something like 12k lines of javascript.